### PR TITLE
Feature/#168/create unlinked and inactive i card page

### PR DESCRIPTION
**Pull request by royayush1**
-Link to Ticket: https://github.com/orgs/UofA-Blueprint/projects/1

- Description
- Changes made to ICard Page on app. Added content to RegistrationView and MyICardView(when status is 'inactive')
- Bulk of content from both views put into shared component called ICardPage
- Most of functionality for both views happens in ICardPage.jsx
- Shared Component called FancyBox.jsx made as a stylistic option for buttons and alerts. Used them in both the views
- Verify Account button made when user signs in and their status is inactive. For now, just a dummy button
- Test made, but not working because of jest issues. The test itself is fine

- Screenshots
- Registration View
WhatsApp Image 2022-10-23 at 3 40 34 PM

-MyICardView when status is inactive
WhatsApp Image 2022-10-23 at 3 40 34 PM (1)

### CODE REVIEW

File: mobile/src/views/RegistrationView.jsx [hunk 1]
```diff
@@ -13,8 +12,10 @@ import AuthContext from '../context/AuthContext';
 WebBrowser.maybeCompleteAuthSession();
 
 import {CLIENT_ID, API_ROUTE, API_KEY} from '@env';
+import MyICardPage from '../components/shared/ICardPage';
 
 const expoClientId = CLIENT_ID;
+console.log(expoClientId);
```
Comment (yismailuofa) [inline mobile/src/views/RegistrationView.jsx]: Remove console log

File: mobile/src/views/MyICardView.jsx [hunk 2]
```diff
@@ -1,204 +1,27 @@
...
-        ) : (
-          <></>
-        )}
-      </View>
-      <Image
-        source={require('../../assets/ISA-logo.png')}
-        style={styles.logo}
-      />
-    </View>
+    <MyICardPage user = {user} status = {finalStatus} verify = {() => {console.log('verify')}}/>
```
Comment (yismailuofa) [inline mobile/src/views/MyICardView.jsx:22]: If this verify is TODO should maybe mark it with a comment

**Review by MoyinF — APPROVED** [review]
So far, everything looks good to me. I tested it and had no issues. I also like the description with the screenshots and the link to the ticket. Just take care of the things Youssef mentioned.

**Comment by royayush1** [comment]
Done with the changes :)

**Pull request merged**